### ISSUE TITLE
Issue when running seed against schema with Photon

### ISSUE THREAD

**Issue by tvvignesh** [prisma/photonjs#270]
**UPDATE:** Was able to figure out the reason for this bug - added it as a comment.

Hi. I tried running some seed data against my schema as mentioned below. And got the error as mentioned in the screenshot below asking me to file an issue here.

![Screenshot from 2019-10-27 12-11-40](https://user-images.githubusercontent.com/1165845/67630777-f3395a00-f8b2-11e9-9ed8-90854977dca3.png)

Schema used:

```
model Phone {
    id            String   @default(cuid()) @id
    alias         String
    contactNo     String
    contactType   String
    settingListID String[]
    tagListID     String[]
    createdDate   DateTime @default(now())
    updatedDate   DateTime @updatedAt
}

model Email {
    id            String   @default(cuid()) @id
    alias         String
    emailAddress  String
    emailType     String
    settingListID String[]
    tagListID     String[]
    createdDate   DateTime @default(now())
    updatedDate   DateTime @updatedAt
}

model User {
    id            String   @default(cuid()) @id
    firstName     String
    lastName      String
    displayName   String
    username      String
    dob           DateTime
    createdDate   DateTime @default(now())
    updatedDate   DateTime @updatedAt
    email         Email[]
    phone         Phone[]
    settingListID String[]
    tagListID     String[]
}
```

Code I am using in seed.js

```
const { Photon } = require('@generated/photon');
const photon = new Photon();
const moment = require('moment');
/**
 * SEED FUNCTION FOR ACCOUNTS TO LOAD INITIAL TEST DATA
 */
async function main() {
    const user1 = await photon.users.create({
        data: {
            firstName: 'Vignesh',
            lastName: 'T.V.',
            username: 'tvvignesh',
            displayName: 'Vignesh',
            dob: moment('1992-12-30').toDate(),
            settingListID: [],
            tagListID: [],
            email: {
                create: [
                    {
                        emailAddress: 'vignesh@timecampus.com',
                        emailType: 'primary',
                        alias: 'primary email',
                        settingListID: [],
                        tagListID: []
                    }
                ]
            },
            phone: {
                create: [
                    {
                        contactNo: '+91-7358445777',
                        alias: 'personal mobile',
                        contactType: 'primary',
                        settingListID: [],
                        tagListID: []
                    }
                ]
            }
        }
    });

    console.log({ user1 });
}

main()
    .catch(e => console.error(e))
    .finally(async () => {
        await photon.disconnect();
    });

```

**Comment by tvvignesh** [comment]
Hi. I did some experimentation and noticed that this issue was caused because I was specifying an empty array declaration for settingListID and tagListID. When I removed both of those fields from the function and ran it again, it worked.

As I understand, even initializing the row with empty list should be valid but it has some issue in unwrapping.

![Screenshot from 2019-10-27 12-27-01](https://user-images.githubusercontent.com/1165845/67630947-372d5e80-f8b5-11e9-9500-a2999bddd79b.png)
